R/arguments.R

```diff
@@ -36,14 +36,14 @@ wsc_parse_args <- function(option_list, mandatory=c()){
 wsc_parse_numeric <- function(opt, varname, val_for_na=NA, length=1){
   if (is.na(opt[[varname]])){
     return(rep(val_for_na, length))
+  }else if(grepl("[0-10]+:[0:10]+",opt[[varname]])){
+        vals <- as.numeric(unlist(strsplit(varname, ":")))
```

Comment:
Don't forget you need the value stored in the list here, not the name of it- so opt[[varname]] rather than just 'varname'